yak/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if(NOT DEFINED CMAKE_CUDA_STANDARD)
   set(CMAKE_CUDA_STANDARD_REQUIRED True)
 endif()
 
-find_package(CUDA 9.0 REQUIRED)
```

Comment:
If you increase the minimum CUDA version to 10 then it'll break compatibility with CUDA 9. We need to maintain compatibility with both CUDA 9 and 10, while eventually adding support for CUDA 11. Conceptually there's no reason why CUDA 9 should not work with Ubuntu 20.04.